src/bulma/pages/products/components/BaseForm.vue

```diff
@@ -1,6 +1,13 @@
 <template>
     <enso-form class="box form-box has-background-light raises-on-hover"
         @ready="form = $event.form">
+        <template v-slot:category_id="{ field }">
+            <label class="label">
+                {{ i18n('Category') }}
+            </label>
+            <categories v-model="field.value"
+                ref="categories"/>
+        </template>
```

Comment:
there's more here to connect .. look in the FormField as example

Reply:
updated